nistats/contrasts.py

```diff
@@ -276,3 +276,71 @@ def __rmul__(self, scalar):
 
     def __div__(self, scalar):
         return self.__rmul__(1 / float(scalar))
+
+
+def fixed_effects_img(contrast_imgs, variance_imgs, mask,
+                         precision_weighted=False):
+    """Compute the fixed effets given images of effects and variance
+
+    Parameters
+    ----------
+    contrast_imgs: list of Nifti1Images or strings
+              the input contrast images
+    variance_imgs: list of Nifti1Images or strings
+              the input variance images
+    mask: Nifti1Image or NiftiMasker instance,
+              mask image
+    precision_weighted: Bool, optional,
+              Whether the fixed effects estimates should be weighted by inverse
+              variance or not. Defaults to False.
+
+    Returns
+    -------
+    ffx_contrast_img: Nifti1Image,
+             the fixed effects contrast computed within the mask
+    ffx_variance_img: Nifti1Image,
+             the fixed effects variance computed within the mask
+    ffx_t_img: Nifti1Image,
```

Comment:
Can other stats be returned like z? or p?

Reply:
Actually, there is a reason why I didn't do it: namely that you would need to provide (additionally) the degrees of freedom for each input, since the dofs of the output is basically the sum of the dofs of inputs. I was not sure it is worth the effort, given that the number of dofs is high in such models, so that eventually t~=z.  WDYT ?

Reply:
Ah, then I agree it's probably not worth it unless someone requests it. I don't personally need it.